### DISCUSSION TITLE
Exclude OS sync

### DISCUSSION THREAD

**Discussion by SerhiiZahuba** [bb-Ricardo/netbox-sync#173] [General]
I have several virtual machines running ROS Mikrotik OS. For them, vcenter is installed
Guest OS: Microsoft Windows Server 2008 R2 (64-bit). How can I exclude OS name synchronization for these virtual machines. Or perhaps you can add a custom OS replacement from Windows Server to ROS or your own name to the config for these machines. Thanks

**Comment by bb-Ricardo** [comment]
Hi,

would this help?:
https://github.com/bb-Ricardo/netbox-sync/blob/d0143119de99fd40e39e923fe111045a85796b96/settings-example.ini#L164-L169

**Comment by SerhiiZahuba** [comment]
No. I have seen this setting. The problem is that there are virtual machines with Microsoft Windows Server 2008 R2 and inside there really is Microsoft Windows Server 2008 R2, but when mikrotik is installed, there is no choice of ROS system in vcenter and vcenter automatically assigns Windows Server 2008 R2 to them.
![image](https://user-images.githubusercontent.com/36307887/162602584-09debe33-fde7-46df-b389-432b43ad18e7.png)
